### PR TITLE
Feature/5796 convert icons to live code examples

### PR DESCRIPTION
**Pull request by komal-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-Convert images to live RN code (using react-native-web)

-Remove unused image assets

-Convert markdown code blocks to PrismJS code blocks

-Use PrismJS to highlight targeted lines in the code example blocks

-Include buttons to copy to clipboard and view full github source

#### Screenshots / Screen Recording (if applicable)

-
Screenshot (20)
Screenshot (21)
Screenshot (22)
Screenshot (23)
Screenshot (24)

#### To Test:

- yarn docs-start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by ektaghag-eaton — APPROVED** [review]

File: docs/src/componentDocs/Icons/markdown/IconsExamples.mdx [hunk 1]
```diff
@@ -12,120 +12,28 @@ Icons are visual indicators usually used to describe action or intent.
...
-      {...props}
-      icon="account-circle"
-      color={'#E5E4E2'}
-    />
-  )}
-/>
-```
+<IconAsInlineFunctionalComponent/>
 
 ## String as a Icon
```
Comment (surajeaton) [inline docs/src/componentDocs/Icons/markdown/IconsExamples.mdx:29]: This should be Icon as a String

**Review by surajeaton — COMMENTED** [review]
You need to run yarn prettier

**Comment by komal-eaton** [comment]
> You need to run yarn prettier

Done

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good

**Pull request merged**